app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    check_command = command.split()
+    if (len(check_command) != 3 or
+            check_command[0] != "cp" or
+            check_command[1] == check_command[2]):
+        return
+    with (open(check_command[1], "r") as file_in,
+          open(check_command[2], "w") as file_out):
+        for line in file_in:
+            file_out.write(line + "\n")
```

Comment:
Adding a newline character with `file_out.write(line + "\n")` is incorrect because `line` already includes the newline character from the input file. You should use `file_out.write(line)` instead to avoid adding extra newlines.